tests/functional/job-submission/20-check-chunking.t

```diff
@@ -20,38 +20,29 @@
 . "$(dirname "$0")/test_header"
 set_test_number 3
 
-create_test_global_config '' '
+create_test_global_config '' "
 [scheduler]
     process pool size = 1
-'
+[platforms]
+    [[$CYLC_TEST_PLATFORM]]
+        max batch submit size = 2
+"
 
-init_suite "${TEST_NAME_BASE}" <<'__FLOW_CONFIG__'
-[scheduler]
-    [[events]]
-        abort on inactivity = True
-        abort on stalled = True
-        inactivity = PT10M
-[task parameters]
-    p = 1..202
```

Comment:
This test used to submit 200 odd jobs to test that the chunking mechanism for job submission was working correctly. This has a significant impact on the test system which was causing some of the other flaky tests to fail in very strange ways (took ages to realise this was causing it).

Made `max batch submit size` configurable to make this test nicer. Also makes this batching a little more transparent and removes an arbitrary hard-coded default so that's nice too.